README.md

```diff
@@ -23,6 +23,7 @@ Boolean options:
          --trap         Trap Ctrl-C and print summary on interrupt.
          --help         Display this help
          --version      Display the version
+		 --loose        Use inconsistencies in subtests in the reort
```

Comment:
This is indented differently than the rest

Reply:
Fixed now.

Reply:
"Use inconsistencies in subtests in the reort" doesn't really make sense to me.

Reply:
Yep... Should say report, but that's a literal interpretation of what the code does. It adds inconsistencies in tests to the final report. Can't really say otherwise. Any suggestion?

Reply:
It ignores all subtests

Reply:
You got it.